diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/identifiers/LocalVariablesRule.kt

```diff
@@ -94,12 +96,28 @@ class LocalVariablesRule(private val configRules: List<RulesConfig>) : Rule("loc
     }
 
     private fun handleConsecutiveDeclarations(statement: PsiElement, properties: List<KtProperty>) {
+        val numLinesAfterLastProp = if (properties.last().node.treeNext.elementType == WHITE_SPACE) {
+            // minus one is needed to except \n after property
+            properties
+                    .last()
+                    .node
+                    .treeNext
+                    .numNewLines() - 1
+        } else {
+            0
+        }
```

Comment:
```suggestion
        val numLinesAfterLastProp =
            properties
                    .last()
                    .node
                    .treeNext
                    .takeIf { it.elementType == WHITE_SPACE }
                    ?.let {
                        // minus one is needed to except \n after property
                        it.numNewLines() - 1 
                    }
                    ?: 0
```
Personally, I'd rewrite it like this, to avoid getting same node two times.